_search-plugins/sql/sql/odbc.md

```diff
@@ -183,13 +181,14 @@ Pre-requisites:
 
 1. Start Tableau. Under the **Connect** section, go to **To a Server** and choose **Other Databases (ODBC)**.
 
-1. In the **DSN drop-down**, select the OpenSearch DSN you set up in the previous set of steps. The options you added will be automatically filled into the **Connection Attributes**.
+2. In the **DSN drop-down**, select the OpenSearch DSN you set up in the previous set of steps. The options you added will be automatically filled into the **Connection Attributes**.
 
-1. Select **Sign In**. After a few seconds, Tableau connects to your OpenSearch server. Once connected, you will directed to  **Datasource** window. The **Database** will be already populated with name of the OpenSearch cluster.
+3. Select **Sign In**. After a few seconds, Tableau connects to your OpenSearch server. Once connected, you will directed to  **Datasource** window. The **Database** will be already populated with name of the OpenSearch cluster.
 To list all the indices, click the search icon under **Table**.
 
-1. Start playing with data by dragging table to connection area. Choose **Update Now** or **Automatically Update** to populate table data.
+4. Start playing with data by dragging table to connection area. Choose **Update Now** or **Automatically Update** to populate table data.
```

Comment:
Change "playing" to "experimenting". Insert "the" before "table" and "connection". Also insert "the" before "table data" (insert all necessary articles).